core/domain/stats_jobs_one_off.py

```diff
@@ -1046,6 +1046,26 @@ def reduce(key, stringified_values):
         yield '%s: %d' % (key, len(stringified_values))
 
 
+class RefreshInteractionRegistryJob(jobs.BaseMapReduceJobManager):
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    # pylint: disable=unused-argument
```

Comment:
I'm also using it to disable unused arguments in reduce(), but I added it to both lines individually instead of using a block disable.